pkg/liquidity-mining/contracts/gauges/StakelessGauge.sol

```diff
@@ -173,4 +177,51 @@ abstract contract StakelessGauge is IStakelessGauge, ReentrancyGuard {
         require(msg.sender == address(_authorizerAdaptor), "SENDER_NOT_ALLOWED");
         _isKilled = false;
     }
+
+    /**
+     * @notice Sets a new maximum relative weight for the gauge.
+     * The value shall be normalized to 1e18, and not greater than ABSOLUTE_MAX_RELATIVE_WEIGHT.
+     * @param maxRelativeWeight New maximum relative weight.
+     */
+    function set_max_relative_weight(uint256 maxRelativeWeight) external override {
+        require(msg.sender == address(_authorizerAdaptor), "SENDER_NOT_ALLOWED");
+        require(
+            maxRelativeWeight <= ABSOLUTE_MAX_RELATIVE_WEIGHT,
+            "Max relative weight exceeds allowed absolute maximum"
+        );
+        max_relative_weight = maxRelativeWeight;
+        emit ILiquidityGauge.MaxRelativeWeightChanged(maxRelativeWeight);
```

Comment:
```suggestion
        emit MaxRelativeWeightChanged(maxRelativeWeight);
```

I think you don't need to declare the interface. I actually didn't know you _could_ declare it

Reply:
Done.